src/constants/url.constant.ts

```diff
@@ -0,0 +1 @@
+export const BASE_URL = 'http://localhost:3000'
```

Comment:
Very nice use of a constant file. Here we should include the paths for endpoints used also. Maybe something like:
````
const API_URLS = {
  PRODUCTS: '/products',
  ...
};
````